README.rst

```diff
@@ -34,12 +36,12 @@ https://github.com/kislyuk/tweak#array-merge-operators for a list of these opera
 
 Service to Service Authorization
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Google service credentials must be whitelisted before they will authenticate with the HCA CLI.
+
+Set the environment variable `GOOGLE_APPLICATION_CREDENTIALS` to the path of your Google service credentials file to
+authenticate.
 
-For HCA/DCP developers using the HCA CLI to connect their service,
```

Comment:
I think this section is useful because it specifies for Dev.

Reply:
This applies to everyone though.  The CLI won't work if you don't have google credentials for ordinary users too.